src/app/static/src/app/store/adr/actions.ts

```diff
@@ -143,20 +144,28 @@ export const actions: ActionTree<ADRState, RootState> & ADRActions = {
         }
     },
 
-    async uploadFilestoADR(context, uploadFilesPayload) {
+    async uploadFilesToADR(context, uploadFilesPayload) {
         const {state, rootState, commit} = context;
+        const uploadMetadata = rootState.modelCalibrate.uploadMetadata
         const selectedDatasetId = rootState.baseline.selectedDataset?.id;
         const modelCalibrateId = rootState.modelCalibrate.calibrateId;
         commit({type: ADRMutation.ADRUploadStarted});
 
         for (let i = 0; i < uploadFilesPayload.length; i++) {
-            const { resourceType, resourceFilename, resourceId } = uploadFilesPayload[i]
+            const { resourceType, resourceFilename, resourceId, displayName } = uploadFilesPayload[i]
 
             const requestParams: Dict<string> = {resourceFileName: resourceFilename}
             if (resourceId){
                 requestParams["resourceId"] = resourceId
             }
 
+            if (displayName === "outputSummary") {
```

Comment:
I know the strings happen to match here, but strictly speaking it might make more sense to compare against `resourceType` rather than `displayName` - the latter is really just an i18n tag.